docgen/src/guide/Refining_hits.md

```diff
@@ -0,0 +1,551 @@
+---
+title: Refining within hits
+mainTitle: Guides
+layout: main.pug
+category: guide
+navWeight: # idk what to put here 🚨
```

Comment:
It's the number that allow you to positionate the guide in the sidebar. If you want it at the end, it should be bigger than the last one (not just by one to let other guide to be in between).